### DISCUSSION TITLE
Transferring nix-ts-mode to nix-community?

### DISCUSSION THREAD

**Discussion by remi-gelinas** [nix-community/infra#1209] [General]
Per https://github.com/remi-gelinas/nix-ts-mode/issues/19, I'd like to get this transferred to the greater Nix community org if it makes sense :)

**Comment by adisbladis** [comment]
I have invited you to the org and created a team. Once you've accepted the invitation you can transfer the repo, and I will add the team to the repo, and you to the team.

**Reply by remi-gelinas** [reply]
Thank you, repo transferred!

**Reply by remi-gelinas** [reply]
@adisbladis Have you been able to add the team and add me to the team? I don't believe I've received an invitation, but the repository has been transferred.

**Reply by adisbladis** [reply]
I had missed that the repo was transferred, but now it's done!

**Reply by remi-gelinas** [reply]
@adisbladis Thank you! I'm just looking through to add some maintainers and branch rules to the repo. Forgive my ignorance, this is the first time I'm really using organizations, but I don't seem to have permissions to add maintainers to the `nix-ts-mode` team? I see myself and Steve Purcell as team members, but in the team overview I am unable to add others, and I am unable to adjust repository settings like branch protection for `nix-community/nix-ts-mode`.

**Reply by Mic92** [reply]
I fixed up all the permissions

**Reply by remi-gelinas** [reply]
Awesome, thanks @Mic92! What is the process for inviting someone to contribute? I think they need to be part of the `nix-community` org to be added to my team, is that an invite I can initiate?

**Reply by Mic92** [reply]
Not sure if you can send an invite. But you can also ask here or in the matrix channel otherwise.
